File: smriti_retail_os/smriti_retail_os/page/smriti_barcode/smriti_barcode.js
// DEPRECATED: Compatibility wrapper for SMRITI Barcode Printer Desk route.
// Canonical implementation resides in: public/js/smriti_barcode.js
// Target removal: SMRITI v2.0

frappe.pages['smriti-barcode'].on_page_load = function(wrapper) {
    frappe.require([
        '/assets/smriti_retail_os/css/smriti-barcode.css',
        '/assets/smriti_retail_os/js/smriti_barcode.js'
    ], function() {
        if (typeof SmritiBarcodeController !== 'undefined') {
            var page = frappe.ui.make_app_page({
                parent: wrapper,
                title: __('SMRITI Barcode Printing'),
                single_column: true
            });
            new SmritiBarcodeController(wrapper, page);
        }
    });
};

File: smriti_retail_os/smriti_retail_os/page/smriti_barcode/smriti_barcode.py
# -*- coding: utf-8 -*-
#
# @file: smriti_retail_os/page/smriti-barcode/smriti-barcode.py
# @description: SMRITI Barcode Frappe page controller.
# @author: Jawahar R Mallah <jawahar.mallah@gmail.com>
# @date: 2026-05-28
# @version: 1.8.6
# @license: MIT
# * Copyright (c) 2026 AITDL NETWORK & ERPNbook.com. All rights reserved.
#

import frappe

def get_page_context(wrapper):
    return {
        "title": "SMRITI Barcode Printing"
    }
